go.mod

```diff
@@ -1 +1,3 @@
 module github.com/lib/pq
+
+go 1.12
```

Comment:
This probably shouldn't go in this change.

Reply:
Removed.